hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtil.java

```diff
@@ -1575,6 +1575,20 @@ public void deleteTable(TableName tableName) throws IOException {
     getAdmin().deleteTable(tableName);
   }
 
+  /**
+   * Drop an existing table
+   * @param tableName existing table
+   * @param archive   if archive the data when delete the table
+   */
+  public void deleteTable(TableName tableName, boolean archive) throws IOException {
```

Comment:
Make the above method call this method so we can save some lines of code?

Reply:
@Apache9  Correct! Will update the code